src/main/java/com/simpligility/maven/plugins/android/phase09package/ApkMojo.java

```diff
@@ -825,7 +825,7 @@ private File removeDuplicatesFromJar( File in, List<String> duplicates,
                     //if not handled by transformer, add (once) to duplicates jar
                     if ( !resourceTransformed )
                     {
-                        if ( !duplicatesAdded.add( entry.getName() ) )
+                        if ( duplicatesAdded.add( entry.getName() ) )
```

Comment:
@andrew-bowley  I think you are correct in that this code block should only be entered `if (duplicatesAdded(.add( entry.getName() )`

But I can't see where `duplicatesAdded` is ever modified, so it always be an emptySet.
@kedzie can you shed some light as this was code you added in version:6a1a483.